src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```
@@ -12,35 +12,40 @@
     <RazorComponent Include="**\*.razor" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
   </ItemGroup>
 
-  <ItemGroup>
-    <!--
-      Web assets (js, css, etc under wwwroot) and .razor files are represented as Content items in the Razor SDK, but run foul of
-      checks in the Android and iOS SDK. This annotates these items with metadata that prevents them from being bundled / examined.
-    -->
-    <Content Update="@(Content->WithMetadataValue('WebContentItem', 'true'))"
-      ExcludeFromContentCheck="true"
-      PublishFolderType="None" />
-
-    <Content Update="@(Content->WithMetadataValue('Extension', '.razor'))"
-      ExcludeFromContentCheck="true"
-      PublishFolderType="None" />
-  </ItemGroup>
-
   <PropertyGroup>
     <StaticWebAssetBasePath>/</StaticWebAssetBasePath>
     <StaticWebAssetProjectMode>Root</StaticWebAssetProjectMode>
   </PropertyGroup>
 
-  <Target Name="ConvertStaticWebAssetsToMauiAssets" BeforeTargets="GetMauiItems;ResizetizeCollectItems" DependsOnTargets="ResolveProjectReferences;StaticWebAssetsPrepareForRun;StaticWebAssetsPrepareForPublish;LoadStaticWebAssetsPublishManifest">
+  <Target Name="ConvertStaticWebAssetsToMauiAssets"
+      BeforeTargets="GetMauiItems;ResizetizeCollectItems;_CollectBundleResources;_CollectHotRestartBundleResources;_CheckForContent"
+      DependsOnTargets="ResolveProjectReferences;StaticWebAssetsPrepareForRun;StaticWebAssetsPrepareForPublish;LoadStaticWebAssetsPublishManifest">
+
     <ComputeStaticWebAssetsTargetPaths Assets="@(StaticWebAsset)" PathPrefix="wwwroot">
       <Output TaskParameter="AssetsWithTargetPath" ItemName="_MauiStaticWebAssetWithTargetPath" />
     </ComputeStaticWebAssetsTargetPaths>
+
     <ItemGroup>
       <MauiAsset Include="%(_MauiStaticWebAssetWithTargetPath.Identity)">
         <Link>%(_MauiStaticWebAssetWithTargetPath.TargetPath)</Link>
         <TargetPath>%(_MauiStaticWebAssetWithTargetPath.TargetPath)</TargetPath>
       </MauiAsset>
     </ItemGroup>
+
+    <!-- Update Content files that are managed by the Razor SDK to opt out of iOS and Android content-checks -->
+    <ItemGroup>
+      <_ContentNotStaticWebAsset Include="@(Content)" Exclude="@(StaticWebAsset)" />
```

I believe this needs to be as below:

```suggestion
      <_ContentNotStaticWebAsset Include="@(Content)" Exclude="%(StaticWebAsset.OriginalItemSpec)" />
```

The reason is that StaticWebAssets always use the full path to the item, and capture the original definition within OriginalItemSpec

---

Turns out we don't need to do this - [RazorSDK removes SWA items from Content](https://github.com/dotnet/sdk/blob/b3bdd3c7567b7f8969aea8404dc0503fcb2690ef/src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets#L668-L678) which makes this a lot easier since we don't have to do set intersections (the list of items that are neither SWA nor ScopedCssInputs).

---

I think we still need to do this for ScopedCssInputs as they get removed from SWA as part of bundling (but I might be wrong)